### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @Eilon - waiting to see if it builds on Travis

**Comment by dougbu** [comment]
Now that this is in, could we make `dev` the default branch in this repo?

File: HtmlAbstractions.sln [hunk 1]
```diff
@@ -0,0 +1,55 @@
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 14
+VisualStudioVersion = 14.0.24711.0
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{A5A15F1C-885A-452A-A731-B0173DDBD913}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "test", "test", "{F31FF137-390C-49BF-A3BD-7C6ED3597C21}"
+EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.AspNet.Html", "src\Microsoft.AspNet.Html.Abstractions\Microsoft.AspNet.Html.xproj", "{68A28E4A-3ADE-4187-9625-4FF185887CB3}"
```
Comment (dougbu) [inline HtmlAbstractions.sln]: Path (`Microsoft.AspNet.Html.Abstractions`) matches the package name everything else expects but name (`Microsoft.AspNet.Html`) matches the actual folders and the package building will create. Was the package rename intended? If yes, solution should be changed to match.

**Comment by Eilon** [comment]
@dougbu it's not in yet, so we can't change the branch. I asked @pranavkm to let me know when it's in because only admins can change the default branch.

File: HtmlAbstractions.sln (hunk 1 above)
Comment (Eilon) [inline HtmlAbstractions.sln]: Hmm yeah that looks messed up... I would think we want to keep the original package name, no?

**Comment by Eilon** [comment]
Ok I take that back, it *is* in, but I guess GitHub is confused because it's a different branch #githubfail. Sorry about that, will change default branch now.

File: HtmlAbstractions.sln (hunk 1 above)
Comment (rynowak) [inline HtmlAbstractions.sln]: Yes. We want to keep the package name, but drop `.Abstractions` from the namespace (similar to almost everything else).

**Comment by pranavkm** [comment]
@dougbu is sending out a PR to fix the sln.

**Pull request closed without merging**